components/compositing/compositor.rs

```diff
@@ -1424,13 +1412,13 @@ impl IOCompositor {
         }
 
         if opts::get().wait_for_stable_image {
-            // The current image may be ready to output. However, if there are animations active,
-            // continue waiting for the image output to be stable AND all active animations to complete.
-            if self.animations_or_animation_callbacks_running() {
-                return Err(UnableToComposite::NotReadyToPaintImage(
-                    NotReadyToPaint::AnimationsActive,
-                ));
-            }
+            //// The current image may be ready to output. However, if there are animations active,
```

Comment:
Can we remove this code instead?

Reply:
Oh, yes. 🤦 Sorry. This is definitely not what I mean to do.